source/AMDSClient.cpp

```diff
@@ -344,6 +345,20 @@ void AMDSClient::requestData()
 			clientMiddleTimePlusCountBeforeAndAfterDataRequest->setCountAfter(countAfter);
 		}
 	}
+	else if(clientRequest->requestType() == AMDSClientRequestDefinitions::Continuous){
+		AMDSClientContinuousDataRequest *clientContinuousDataRequest = qobject_cast<AMDSClientContinuousDataRequest*>(clientRequest);
+		if(clientContinuousDataRequest){
+			quint64 updateInterval = 500;
```

Comment:
Is there a reason why this is hard-coded?  If so, why are we using a quint64 to hold the number 500?